openstack-helm/trilio-openstack/values_overrides/ingress.yaml

```diff
@@ -2,8 +2,8 @@ network:
   wlm_api:
     ingress:
       classes:
-        namespace: "openstack-ingress-nginx"
+        namespace: "nginx"
```

Comment:
Please note this namespace name. I think in MOSK it's 'openstack-ingress-nginx' and in openstack helm it's 'nginx'.
We need to make this configurable through values.yaml file